src/log_configuration.rs

```diff
@@ -0,0 +1,118 @@
+use super::{Buffer, TagMode};
+
+use env_logger::filter::Filter;
+
+/// Documentation here
+pub struct LogConfiguration {
```

Comment:
This is the main API exposed for setting parameters. Can we name this just `Logger`?

Reply:
For my opinion name "Logger"  does not fully reflect the meaning of this class, or?